dynamics/rolldowndata/rolldown/cleaned_RollDown2019-06-02-1.csv

```diff
@@ -0,0 +1,219 @@
+Time Offset [ms],voltage,current,vehicle_velocity_left,vehicle_velocity_right,average_velocity
+9257.388,,,,,
```

Comment:
This is really close, just get rid of all of the ones without a velocity reading :)